profiles/serializers.py

```diff
@@ -34,17 +31,34 @@ def get_is_saved(self, obj):
 
 
 class ProfileDetailSerializer(serializers.ModelSerializer):
-    activity = ActivitySerializer(many=True, read_only=True)
-    category = CategorySerializer(many=True, read_only=True)
+    is_saved = serializers.SerializerMethodField()
+    email = serializers.ReadOnlyField(source='person.email')
```

Comment:
It's a serializer with contacts being hidden, right? Why do we have here `email` field? I thought it supposed to be hidden :)

Reply:
thank you) it was fixed in my new commit